node/src/test/scala/coop/rchain/node/mergeablity/ComputeMerge.scala

```diff
@@ -61,6 +61,7 @@ trait ComputeMerge {
       .use {
         case (runtime, _, historyRepo) =>
           for {
+            _           <- runtime.preGenesisStateHash
```

Comment:
Hm... I'm not sure what this is supposed to do?

Reply:
Insert neccessary part for registry working

Reply:
This  is not neccessary for basic mergebility test. It is only used for treeHashMao tests. I would come up with something else later.

Reply:
I mean, it looks weird.
Does this call invokes some effects on runtime and updates its internal state? As output is ignored. 
And also it is called `preGenesisStateHash ` so I expect it to just give me stateHash. Maybe rename to `initRegistry`?